neuro_data/static_images/data_schemas.py

```diff
@@ -689,11 +740,80 @@ def run_stats(selector, types, ix, axis=None):
             )
             return ret
 
+        def run_stats_input(selector, types, ix, axis=None, per_input=False):
```

Comment:
NOTE: If per_input is set to True or the inputs contain at least one masked image, population standard deviation is reported instead of sample standard deviation.